src/Exception/DocraptorException.php

```diff
@@ -0,0 +1,5 @@
+<?php namespace DocRaptor\Exception;
+
+interface DocRaptorException
```

Comment:
Is this used anywhere? Seems like the other exceptions should extend/implement? Even if this is a blank interface